src/libraries/tests.proj

```diff
@@ -68,6 +68,11 @@
     <ProjectExclusions Include="$(MSBuildThisFileDirectory)System.Text.RegularExpressions\tests\System.Text.RegularExpressions.Tests.csproj" />
   </ItemGroup>
 
+  <ItemGroup Condition="'$(TargetOS)' == 'Android' and '$(TargetArchitecture)' == 'x86'">
+    <!-- https://github.com/dotnet/runtime/issues/50493 -->
+    <FunctionalTestExclusions Include="$(RepoRoot)\src\tests\FunctionalTests\Android\Device_Emulator\AOT\Android.Device_Emulator.Aot.Test.csproj" />
```

Comment:
Curious why didn't we choose to use `ProjectExclusions` for this case?

Reply:
Hmm it felt like a good idea to separate from the normal unit tests but now that I look again it's probably unnecessary. Will merge to unblock the tests and rename this in a follow-up PR.